src/test/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResourceBaseIntegrationTest.java

```diff
@@ -20,7 +20,9 @@
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public abstract class CollectionResourceBaseIntegrationTest extends CqlEnabledIntegrationTestBase {
```

Comment:
The namespace name is not easy to change as it depends on the `CqlEnabledIntegrationTestBase`.. I think we should go away from the CQL based tests, as now we have all the commands to run the tests purely using jsonapi commands.. Should I adapt that in this PR?

Reply:
If it's simple enough to do, yes, I think that makes sense.